mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -157,9 +197,9 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         submitDOM.attr('disabled', 'disabled');
 
         /* Something went wrong with student submission, denied next question */
-        if (result.step != active_child+1) {
-            active_child = result.step-1;
-            displayNextChild();
+        if (response.step != active_child+1) {
+            active_child = response.step-1;
+            // displayNextChild();
```

Comment:
Was this commented out on purpose? Prefer to delete it if it's no longer needed.